src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayRent = 40;
+
+  if (days < 3) {
+    return dayRent * days;
+  } else if (days < 7
+    && days >= 3) {
+    return dayRent * days - 20;
+  } else {
+    return dayRent * days - 50;
```

Comment:
according to point 3 in the checklist, maybe it is better not to use 'else' for the last 'return'? https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md